packages/salesforcedx-sobjects-faux-generator/src/types/index.ts

```diff
@@ -8,9 +8,53 @@
 export {
   BatchRequest,
   BatchResponse,
-  SObject,
+  ChildRelationship,
   Field,
-  ChildRelationship
+  SObject
 } from './describe';
-
 export { SObjectCategory, SObjectRefreshSource } from './general';
+
+import { SObjectShortDescription } from '../describe';
+import { SObject } from './describe';
+import { SObjectCategory, SObjectRefreshSource } from './general';
+
+export interface FieldDeclaration {
```

Comment:
I would move everything below this line into one of the other files in this module and reference them here for exporting.